nservicebus/hosting/index.md

```diff
@@ -27,6 +27,8 @@ There are several approaches to hosting.
  * [Endpoint Lifecycle](/nservicebus/lifecycle/)
  * [Critical Error handling](critical-errors.md)
 
+NOTE: Be aware of critical errors, not configuring a critical error callback makes an endpoints unresponsive after a critical error without terminating the process.
```

Comment:
Are we documenting a bug here? Should we force users to always provide the custom implementation?